spec/factories/admin_users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :admin_user do
+    email { Faker::Internet.email }
```

Comment:
We've had problems before with Faker generating non-unique values in a single run of the test suite. To avoid this, we've been appending a sequential number to any Faker-generated attributes that must be unique. See the partners factory, e.g.